loadtest/control/actions.go

```diff
@@ -960,16 +961,16 @@ func ReloadGQL(u user.User) UserActionResponse {
 			}
 		}
 
-		ver, err := u.Store().ServerVersion()
+		serverVersionString, err := u.Store().ServerVersion()
```

Comment:
Do we still support graphql?

Reply:
Ah, great point! Let me tackle this in a different PR